src/generate/helpers/reduxBindings/writeOpenapiJsonFile.ts

```diff
@@ -0,0 +1,38 @@
+import { camelize } from '@rvoh/dream'
+import * as fs from 'node:fs/promises'
+import * as path from 'node:path'
+import psychicPath from '../../../helpers/path/psychicPath.js'
+
+export default async function writeOpenapiJsonFile({
+  exportName,
+  schemaFile,
+  apiFile,
+  apiImport,
+  outputFile,
+}: {
+  exportName: string
+  schemaFile: string
+  apiFile: string
+  apiImport: string
+  outputFile: string
+}) {
+  const destFilename = `${camelize(exportName)}.openapi-codegen.json`
+  const destDir = path.join(psychicPath('conf'), 'openapi')
+  const destPath = path.join(destDir, destFilename)
+
+  const jsonData = {
+    schemaFile: path.join('..', '..', '..', schemaFile),
```

Comment:
This is written into a file that is then used at a later time to find files from within Typescript, right? So these files shouldn't be `path.join`ed, only at the time when they are read (which, if we're passing them into rtk, it should convert to each system automatically).